index.html

```diff
@@ -4,263 +4,310 @@
-                              </div>
-                              <ul class="skillul">
-                              </ul>
-                          </div>
-                          <div class="listfram ">
-                              <div class="listtag ">
-                          <button type="button" class="icon"><img src="./images/Frame.svg" alt="button"></button>
-                              </div>
-                              <ul class="skillul">
-                              </ul>
-                          </div>
-                
-                      </div>
-                  </div>
-                </section>
+                        <div class="listtag ">
+                            <button type="button" class="icon"><img src="./images/F1.svg.svg" alt="button2"></button>
+                        </div>
+                        <ul class="skillul">
+                        </ul>
+                    </div>
+                    <div class="listfram ">
+                        <div class="listtag ">
+                    <button type="button" class="icon"><img src="./images/Frame.svg" alt="button"></button>
+                        </div>
+                        <ul class="skillul">
+                        </ul>
+                    </div>
+                    </div>
+                    </div>
+                  </section>
 
-
-
-                <!--Contact info section-->
+              <!--Contact info section-->
               <section class="contact-info">
                 <article class="contact-headings"> 
                   <h2>Contact me</h2>
-                    <p class="contact-details">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p> 
+                   <p> If you have an application you are interested in developing, 
+                     a feature that you need built or a project that needs coding.
+                      I’d love to help with it  </p> 
                   </article>
 
                 <form action="https://formspree.io/f/xgeddrdw" method="POST">
 
-                    <input type="text" name="username" id="name" placeholder="Your name" maxlength="30" required/>
-                   
-                    <input type="email" name="email" id="email" placeholder="Your email" required/>
+                <input type="text" name="username"  maxlength="30" placeholder="Full name" required>
+
+                <input type="email" name="email" placeholder="Email" required>
 
                 <textarea name="message" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
-                <button class="form-submit" type="submit">Get In Touch </button>
+                <button type="submit" class="click" value="Submit">Get In Touch</button>
```

Comment:
- [ ] You've done an incredible job to apply the requirement of the project, but you also have to match your design to the one provided from figma design.  Your buttons seemed to be position in wrong place. please fix it to match to the provided design. 
![image](https://user-images.githubusercontent.com/43877689/172958959-94d8497a-56bd-4e0c-85a1-5a00513b0e5b.png)

![image](https://user-images.githubusercontent.com/43877689/172959059-4b8fa618-ddfb-4a3d-a87e-e543e1265900.png)